src/components/Filters/FilterChecklist.vue

```diff
@@ -34,35 +34,33 @@
         :key="index"
         class="filter-checkbox-wrapper"
       >
-        <!--eslint-disable vuejs-accessibility/label-has-for -->
-        <label class="checkbox" :disabled="isDisabled(item)">
-          <input
-            :key="index"
-            :name="item.code"
-            type="checkbox"
-            class="filter-checkbox me-2"
-            :checked="item.checked"
-            :disabled="isDisabled(item)"
-            @change="onValueChange"
-          />
+        <Checkbox
+          :id="item.code"
+          :key="index"
+          class="filter-checkbox me-2"
+          :checked="item.checked"
+          :disabled="isDisabled(item)"
+          @change="onValueChange"
+        >
           <LicenseIcons v-if="filterType === 'licenses'" :license="item.code" />
-          {{ itemLabel(item) }}
-        </label>
-        <button class="appearance-none" type="button">
+          {{ itemLabel(item) }}</Checkbox
+        >
```

Comment:
Same thing here with the whitespace, it doesn't affect the output so we can put the Checkbox closing tag all on one line to make it easier to find compared to the opening tag.

```suggestion
          {{ itemLabel(item) }}
        </Checkbox>
```